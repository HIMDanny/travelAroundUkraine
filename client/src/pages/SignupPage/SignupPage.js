import React from 'react';
import { Container, Typography } from '@mui/material';
import { SignupForm } from '../../features/signup/components';

const SignupPage = () => (
  <Container sx={{ mt: '200px', mb: '300px' }}>
    <Typography variant="h2" align="center" sx={{ mb: '20px' }}>
      Sign Up
    </Typography>
    <SignupForm />
  </Container>
);

export default SignupPage;
