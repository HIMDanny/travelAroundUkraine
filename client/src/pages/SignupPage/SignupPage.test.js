/* eslint-disable react/prop-types */
import { render } from '@testing-library/react';
import SignupPage from './SignupPage';

jest.mock('@mui/material/Container', () => ({ children }) => <div>{children}</div>);
jest.mock('@mui/material/Typography', () => ({ children }) => <p>{children}</p>);
jest.mock('../../features/signup/components/SignupForm/SignupForm', () => () => <div>SIGN UP FORM</div>);

describe('SignupPage snapshot test', () => {
  test('should SignupPage match snapshot', () => {
    const { asFragment } = render(<SignupPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
