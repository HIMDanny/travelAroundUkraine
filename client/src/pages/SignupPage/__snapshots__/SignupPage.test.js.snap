// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignupPage snapshot test should SignupPage match snapshot 1`] = `
<DocumentFragment>
  <div>
    <p>
      Sign Up
    </p>
    <div>
      SIGN UP FORM
    </div>
  </div>
</DocumentFragment>
`;
